CHANGES.rst

```diff
@@ -6,11 +6,19 @@ Unreleased
 ==========
 
 - Provide a default ``config.yml`` in the Docker image, which can be replaced
-  by mounting a file on ``/etc/crate_adapter/config.yml``.
+  by mounting a file on ``/etc/cratedb-prometheus-adapter/config.yml``.
 
 - Updated project to make use of `Go modules <https://golang.org/ref/mod>`_
   instead of Govendor.
 
```

Comment:
I'd suggest that we either create a   `BREAKING CHANGES` subsection or prepend the bellow entries with this to make these breaking changes more visible.

Reply:
Thank you. I've added an admonition with ded226578555f.